packages/portal/src/app/rentalpage/rentalpage.component.scss

```diff
@@ -168,3 +168,35 @@
     }
   }
 }
+
+@media (max-width: 480px){
+
```

Comment:
When coding mobile first, we typically don't need to add the mobile devices media query widths, since they're the default, and only the largest viewports become the overrides. 
One last change I would request is trying adjust to that so we comply with best practices.

Reply:
Okay, I'll try that.
Thanks